library/src/main/java/org/apache/apex/malhar/lib/dedup/BoundedDedupOperator.java

```diff
@@ -156,8 +156,7 @@ protected Slice getKey(Object tuple)
   protected void putManagedState(Object tuple)
   {
     Slice key = getKey(tuple);
-    ((ManagedTimeStateImpl)managedState).put(getBucketId(key), DEFAULT_CONSTANT_TIME,
-        key, new Slice(new byte[0]));
+    ((ManagedTimeStateImpl)managedState).put(getBucketId(key), DEFAULT_CONSTANT_TIME, key, new Slice(null, 0, 0));
```

Comment:
Apart from the correct usage of Slice, APEXMALHAR-2492 is also the set of changes required to ensure support for newer netlet library (>= 1.3.0) which is used by apex-core 3.6.0.
The above change is the minimum required change since build with apex-core 3.6.0/netlet 1.3.0 fails without this change.